src/_includes/training_hero.liquid

```diff
@@ -0,0 +1,24 @@
+<section class="training-hero">
+    <div class="training-hero__background-image" style="background-image: url({{ site.baseurl }}{{ page.intro.background-image }})"> </div>
+    <div class="training-hero__lower-background-block"></div>
```

Comment:
I'm not sure this will need a separate div just for the background. I think we can re-organise the structure of this to be a bit cleaner and possibly use fewer elements.